### PR TITLE
chore(shortint): update multi-bit gpu parameters set

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs [hunk 1]
```diff
@@ -75,85 +74,85 @@ pub const PARAM_GPU_MULTI_BIT_GROUP_2_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64:
...
     )),
     glwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
         2.845267479601915e-15,
     )),
     pbs_base_log: DecompositionBaseLog(22),
     pbs_level: DecompositionLevelCount(1),
-    ks_base_log: DecompositionBaseLog(3),
-    ks_level: DecompositionLevelCount(5),
+    ks_base_log: DecompositionBaseLog(5),
+    ks_level: DecompositionLevelCount(3),
```
Comment (agnesLeroy) [inline tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs:126]: Did you use the GPU cost model to generate the parameters? It seems strange that the KS levels are reduced and n increases :thinking:

Comment (agnesLeroy) [reply tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs:126]: Btw I'm running the multi-bit fast benches to check the effect on performance.

Comment (agnesLeroy) [reply tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs:126]: It worsens performance a bit

Comment (agnesLeroy) [reply tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs:126]: cc @samcrx

Comment (soonum) [reply tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs:126]: Yes I've used it. The one with the values for 256bits chunks

**Review by agnesLeroy — APPROVED** [review]

**Pull request merged**